src/pageObject.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
-class PageObject {
-
+class PageObject extends Header {
```

Comment:
PageObject should not extend Header. Instead, it should have a header property that is an instance of the Header class. Change the inheritance to composition 🤔